website/static/js/osfHelpers.js

```diff
@@ -357,6 +358,57 @@ ko.bindingHandlers.anchorScroll = {
 };
 
 /**
+ * Adds class returned from iconmap to the element. The value accessor should be the 
+ * category of the node.
+ * Example:
+ * <span data-bind="getTheIcon: 'analysis'"></span>
+ */
+ko.bindingHandlers.getTheIcon = {
+    init: function(elem, valueAccessor) {
+        var icon;
+        var category = valueAccessor();
+        if (Object.keys(iconmap.componentIcons).indexOf(category) >=0 ){
+            icon = iconmap.componentIcons[category];
+        }
+        else {
+            icon = iconmap.projectIcons[category];
+        }
+        $(elem).addClass(icon);
+    },
+    update: function(elem, valueAccessor) {
```

Comment:
That makes sense, but I just tried it out locally and the icon does not update if a second contributor changes the category while I am viewing the project. Mind looking into this?